lambda/app.js

```diff
@@ -1,43 +1,48 @@
 'use strict';
 
-var dxf = require('dxf'),
-  request = require('request');
+const dxf = require('dxf');
+const request = require('request');
 
-exports.handler = function (event, context) {
+exports.handler = (event, context) => {
   const body = JSON.parse(event.body);
 
-  request({ url: body.url, encoding: null }, function (error, response, body) {
-    var dxfContents = body.toString();
-    var units = determineUnits(dxfContents);
-    var svgSTR = dxf.toSVG(dxf.parseString(dxfContents));
-    svgSTR = svgSTR.replace("INSUNITS", units);
-    svgSTR = svgSTR.replace("INSUNITS", units);
-    context.done(null, { svg: svgSTR });
+  request({ url: body.url, encoding: null }, (error, response, body) => {
+    const dxfContents = body.toString();
+
+    const outputSVG = dxf
+      .toSVG(dxf.parseString(dxfContents))
+      .replace("INSUNITS", determineUnits(dxfContents));
+
+    context.done(
+      null,
+      {
+        isBase64Encoded: false,
+        statusCode: 200,
+        headers: { "Access-Control-Allow-Origin": "*" },
+        body: JSON.stringify({ "svg": outputSVG })
+      }
+    );
   });
 };
 
 function determineUnits(dxfContents) {
-  var unitSVG = "INSUNITS";
+  let unitSVG = "INSUNITS";
+
   if (dxfContents.includes("$INSUNITS")) {
-    var splitOnUnits = dxfContents.split("$INSUNITS");
-    var splitOnNine = splitOnUnits[1].split("9");
-    var splitOnSeventy = splitOnNine[0].split("70");
-    var units = splitOnSeventy[1].replace(/(\r\n\t|\n|\r\t)/gm, "");
-    var unit = parseInt(units);
+    const splitOnUnits = dxfContents.split("$INSUNITS");
+    const splitOnNine = splitOnUnits[1].split("9");
+    const splitOnSeventy = splitOnNine[0].split("70");
+    const units = splitOnSeventy[1].replace(/(\r\n\t|\n|\r\t)/gm, "");
```

Comment:
I know this is mostly a formatting change, but :ribbon: I think including a comment with an example would make this parsing code easier to understand. Reverse-engineering the code I think we expect something like this, although I'm not 100% sure, and the meaning of these fields is not very intuitive:

```
blah $INSUNITS blah 70
	cm
9 blah
```

Reply:
I agree that some clearer direction explaining why we are splitting the strings and what the number in the end means would be helpful, but since I was trying to keep the reformatting to a minimum I have opted not to redo this code too heavily. I think in the long run if we put more time into this app I would like to see this section refactored into code that's clearer to understand.